### DISCUSSION TITLE
How to enable device plugin feature gate after k3s in installed

### DISCUSSION THREAD

**Discussion by rolfberkenbosch** [k3s-io/k3s#4596] [Q&A]
Hi all,

How can i enable device plugin feature gate after k3s is installed ?
I have tried to modify the k3s.service file with adding the --kubelet-arg="feature-gates=DevicePlugins=true" in the ExecStart field. But no luck.

Anyone have any idea ?

**Comment by brandond** [comment]
`--kubelet-arg=feature-gates=DevicePlugins=true` should work, assuming you don't need to enable the feature-gate on any other components (such as the controller-manager or scheduler). You will of course also need to do a `systemctl daemon-reload` and restart the K3s service after changing the args in the unit file.
